src/icons/GemWithArc.tsx

```diff
@@ -13,13 +13,13 @@ const textStyle = classnames(
   fill('fill-tertiary'),
   textTransform('uppercase'),
   fontWeight('font-semibold'),
-  fontSize('text-3xl', 'md:text-xl')
+  fontSize('text-3xl', 'md:text-xl'),
+  willChange('will-change-transform')
 )
 const svgBox = classnames(
   height('md:h-80', 'h-44'),
   margin('mx-auto', '-mb-4'),
-  scale('scale-200'),
-  willChange('will-change-auto')
+  scale('tiny:scale-175')
```

Comment:
Fixes this clipped text
![Icz7d0Syna](https://user-images.githubusercontent.com/49658988/174143509-fc484eb8-99e8-4e54-8d42-b4e552e41a80.png)

Reply:
Fixed positioning for this as well